src/us/kbase/typedobj/test/files/SubsetAndMetadataExtraction/KB.spec

```diff
@@ -41,8 +38,6 @@ module KB {
 	} FloatStructure;
 
 	/*
-	  @searchable ws_subset numbers
```

Comment:
Since we still have the @searchable ws_subset annotation in specs in production now, I think we should keep at least one test case with this annotation in place to ensure that having it in the parsed structure doesn't cause an error in metadata extraction.